docs/pages/versions/unversioned/sdk/notifications.md

```diff
@@ -856,6 +888,8 @@ A `Promise` resolving to the channel object (of type [`NotificationChannel`](#no
 
 Assigns the channel configuration to a channel of a specified name (creating it if need be). This method lets you assign given notification channel to a notification channel group.
 
+> **Note:** For some settings to be applied on all Android versions it may be necessary to duplicate the configuration between a single notification and notification channel. For example for a notification to play a custom sound on Android versions below 8.0, the custom notification sound has to be set on the notification and for the custom sound to play on Android versions above 8.0, the notification channel that the notification will be assigned to has to have the custom sound configured (for more information see [Setting custom notification sound on Android](#setting-custom-notification-sound-on-android) instruction).
```

Comment:
```suggestion
> **Note:** For some settings to be applied on all Android versions, it may be necessary to duplicate the configuration across both a single notification _and_  it's respective notification channel. For example, for a notification to play a custom sound on Android versions **below** 8.0, the custom notification sound has to be set on the notification (through the [`NotificationContentInput`](#notificationcontentinput)), and for the custom sound to play on Android versions **above** 8.0, the relevant notification channel must have the custom sound configured (through the [`NotificationChannelInput`](#notificationchannelinput)). For more information, see ["Setting custom notification sounds on Android"](#setting-custom-notification-sounds-on-android).
```